packages/scss/src/components/formLabel/component.scss

```diff
@@ -0,0 +1,20 @@
+@mixin component($atRoot: 'without: rule') {
+  color: var(--components-formLabel-color);
+  display: inline-block;
```

Comment:
Maybe `display: inline-flex` + `gap` + `align-items` here ? For consistency with others similars components

Reply:
That's the first way I tried, but even by playing with wrap I did not success to have a good line breaking management, including asterisk and "?" icon.